### PR TITLE
PP-446 update bundled packages

### PR DESCRIPTION
**Pull request by pkuiper-ultimaker**
Update the list of bundled materials with Cura.

### CODE REVIEW

File: resources/bundled_packages/cura.json [hunk 1]
```diff
@@ -1714,7 +1732,7 @@
             "description": "Example package for material and quality profiles for Ultimaker materials.",
             "package_version": "1.4.0",
             "sdk_version": "8.6.0",
-            "website": "https://ultimaker.com/products/materials/cpe",
+            "website": "https://ultimaker.com/in/cura/materials/ultimaker-cpe-plus/printing-guidelines",
```
Comment (saumyaj3) [inline resources/bundled_packages/cura.json:1735]: Isn't this support website?
https://ultimaker.com/materials/s-series-cpe-plus/

File: resources/bundled_packages/cura.json [hunk 2]
```diff
@@ -1828,33 +1903,128 @@
...
-            "website": "https://ultimaker.com/products/materials/tough-pla",
+            "website": "https://ultimaker.com/materials/method-series-abs-carbon-fiber/",
             "author": {
                 "author_id": "UltimakerPackages",
                 "display_name": "UltiMaker",
                 "email": "materials@ultimaker.com",
                 "website": "https://ultimaker.com",
                 "description": "Professional 3D printing made accessible.",
-                "support_website": "https://ultimaker.com/en/resources/troubleshooting/materials"
+                "support_website": "https://support.makerbot.com/s/materials/method"
```
Comment (saumyaj3) [inline resources/bundled_packages/cura.json:1932]: I
```suggestion
                "support_website": "https://support.ultimaker.com/s/article/How-to-print-with-Method-ABS-CF"
```

File: resources/bundled_packages/cura.json [hunk 3]
```diff
@@ -1828,33 +1903,128 @@
...
+            "package_version": "2.0.0",
+            "sdk_version": "8.6.0",
+            "website": "https://ultimaker.com/materials/method-series-abs-r/",
+            "author": {
+                "author_id": "UltimakerPackages",
+                "display_name": "UltiMaker",
+                "email": "materials@ultimaker.com",
+                "website": "https://ultimaker.com",
+                "description": "Professional 3D printing made accessible.",
+                "support_website": "https://support.makerbot.com/s/materials/method"
```
Comment (saumyaj3) [inline resources/bundled_packages/cura.json:1951]:
```suggestion
                "support_website": "https://support.ultimaker.com/s/article/How-to-print-with-Method-ABS-R"
```

File: resources/bundled_packages/cura.json [hunk 4]
```diff
@@ -1828,33 +1903,128 @@
...
+            "package_version": "2.0.0",
+            "sdk_version": "8.6.0",
+            "website": "https://ultimaker.com/materials/method-series-asa/",
+            "author": {
+                "author_id": "UltimakerPackages",
+                "display_name": "UltiMaker",
+                "email": "materials@ultimaker.com",
+                "website": "https://ultimaker.com",
+                "description": "Professional 3D printing made accessible.",
+                "support_website": "https://support.makerbot.com/s/materials/method"
```
Comment (saumyaj3) [inline resources/bundled_packages/cura.json:1970]:
```suggestion
                "support_website": "https://support.ultimaker.com/s/article/How-to-print-with-Method-ASA"
```

File: resources/bundled_packages/cura.json [hunk 5]
```diff
@@ -1828,33 +1903,128 @@
...
+            "package_version": "2.0.0",
+            "sdk_version": "8.6.0",
+            "website": "https://ultimaker.com/materials/method-series-nylon-12-carbon-fiber/",
+            "author": {
+                "author_id": "UltimakerPackages",
+                "display_name": "UltiMaker",
+                "email": "materials@ultimaker.com",
+                "website": "https://ultimaker.com",
+                "description": "Professional 3D printing made accessible.",
+                "support_website": "https://support.makerbot.com/s/materials/method"
```
Comment (saumyaj3) [inline resources/bundled_packages/cura.json:1989]:
```suggestion
                "support_website": "https://support.ultimaker.com/s/article/How-to-print-with-Method-Nylon12-CF"
```

File: resources/bundled_packages/cura.json [hunk 6]
```diff
@@ -1828,33 +1903,128 @@
...
+            "package_version": "2.0.0",
+            "sdk_version": "8.6.0",
+            "website": "https://ultimaker.com/materials/method-series-rapidrinse/",
+            "author": {
+                "author_id": "UltimakerPackages",
+                "display_name": "UltiMaker",
+                "email": "materials@ultimaker.com",
+                "website": "https://ultimaker.com",
+                "description": "Professional 3D printing made accessible.",
+                "support_website": "https://support.makerbot.com/s/materials/method"
```
Comment (saumyaj3) [inline resources/bundled_packages/cura.json:2008]:
```suggestion
                "support_website": "https://support.ultimaker.com/s/article/How-to-print-with-Method-RapidRinse"
```

File: resources/bundled_packages/cura.json [hunk 7]
```diff
@@ -1828,33 +1903,128 @@
...
+            "package_version": "2.0.0",
+            "sdk_version": "8.6.0",
+            "website": "https://ultimaker.com/materials/method-series-sr-30/",
+            "author": {
+                "author_id": "UltimakerPackages",
+                "display_name": "UltiMaker",
+                "email": "materials@ultimaker.com",
+                "website": "https://ultimaker.com",
+                "description": "Professional 3D printing made accessible.",
+                "support_website": "https://support.makerbot.com/s/materials/method"
```
Comment (saumyaj3) [inline resources/bundled_packages/cura.json:2027]:
```suggestion
                "support_website": "https://support.ultimaker.com/s/article/How-to-print-with-Method-SR-30"
```

**Review by saumyaj3 — CHANGES_REQUESTED** [review]
I found that links of the makerbot materials were not properly provided. Can you fix this?

File: resources/bundled_packages/cura.json (hunk 1 above)
Comment (pkuiper-ultimaker) [reply resources/bundled_packages/cura.json:1735]: Correct, I'll change that.

**Pull request merged**